MantidQt/CustomInterfaces/test/ReflDataProcessorPresenterTest.h

```diff
@@ -0,0 +1,524 @@
+#ifndef MANTID_CUSTOMINTERFACES_REFLDATAPROCESSORPRESENTERTEST_H
+#define MANTID_CUSTOMINTERFACES_REFLDATAPROCESSORPRESENTERTEST_H
+
+#include <cxxtest/TestSuite.h>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "MantidAPI/FrameworkManager.h"
+#include "MantidAPI/TableRow.h"
+#include "MantidDataObjects/EventWorkspace.h"
+#include "MantidQtCustomInterfaces/Reflectometry/ReflGenericDataProcessorPresenterFactory.h"
+#include "MantidQtMantidWidgets/DataProcessorUI/DataProcessorMockObjects.h"
+#include "MantidQtMantidWidgets/DataProcessorUI/ProgressableViewMockObject.h"
+#include "MantidTestHelpers/WorkspaceCreationHelper.h"
+
+using namespace Mantid::API;
+using namespace Mantid::Kernel;
```

Comment:
Not needed:
`using namespace Mantid::Kernel;`
`using namespace Mantid::DataObjects;`
`using namespace MantidQt::MantidWidgets;`